Hymns/Display/DisplayHymnView.swift

```diff
@@ -22,19 +22,23 @@ struct DisplayHymnView: View {
                 Text(viewModel.title)
                     .fontWeight(.semibold)
                 Spacer()
+                if viewModel.title.isEmpty {
+                    EmptyView()
+                } else {
                 Button(action: {self.viewModel.toggleFavorited()
```

Comment:
ohh great find! I was wondering why i got an empty favorite sometimes

Could we put this logic in the View Model instead? We can put the fetchFavoriteStatus call inside of the receiveValue, then we won't fetch the favorites until after the hymn come back

Reply:
So, I tried a few things. But as soon as I stopped using the viewModel.title there was always a lack of sync between when the title would come in and the heart toggle. It is split seconds but still noticeable. As long as I leave it viewModel.title it is smooth and they come in at the exact same time the title and the favorites heart. Idk, you can take a stab at it if you want but this was my experience.